### PR TITLE
Add some linking for hw vids

### PR DESCRIPTION
**Pull request by ganicke**
- [x] Put links for HW vids in the device and reference pages.
- [x] Fix the videos gallery/card page so double images wont show in [doc view](https://makecode.microbit.org/behind-the-makecode-hardware)
- [x] Add youtube urls to videos gallery/card page so linking works in doc view.
- [x] Add temperature sensing to _device.md_.
- [x] Remove old cards from LED ref page.

### CODE REVIEW

File: docs/behind-the-makecode-hardware.md [hunk 1]
```diff
@@ -10,31 +10,32 @@
         "name": "LEDs",
         "description": "See how these amazing little devices create light in this fun and illuminating video.",
         "youTubeId": "qqBmvHD5bCw",
-        "imageUrl": "/static/mb/behindhardware/leds.jpg"
```
Comment (pelikhan) [inline docs/behind-the-makecode-hardware.md]: youtube is block in some schools, we keep a local copy of the image on our side.

File: docs/reference/input/acceleration.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ input.acceleration(Dimension.X);
 You measure acceleration with the **milli-g**, which is 1/1000 of a **g**.
 A **g** is as much acceleration as you get from Earth's gravity.
 
+Watch this [video](https://www.youtube.com/watch?v=byngcwjO51U) to learn how the accelerometer works.
```
Comment (pelikhan) [inline docs/reference/input/acceleration.md]: You can simply put the youtube link on a line on the player will be inlined.

File: docs/reference/input/button-is-pressed.md [hunk 3]
```diff
@@ -33,6 +33,12 @@ basic.forever(() => {
 })
 ```
 
+## ~ hint
+
+Find out how buttons provide input to the @boardname@ in this [video](https://www.youtube.com/watch?v=t_Qujjd_38o).
```
Comment (pelikhan) [inline docs/reference/input/button-is-pressed.md]: inline video player.

File: docs/reference/input/light-level.md [hunk 4]
```diff
@@ -43,6 +43,12 @@ basic.forever(() => {
 })
 ```
 
+## ~ hint
+
+Learn more about how light level is detected in this [light sensor](https://www.youtube.com/watch?v=TKhCr-dQMBY) video.
```
Comment (pelikhan) [inline docs/reference/input/light-level.md]: inline video player

File: docs/behind-the-makecode-hardware.md (hunk 1 above)
Comment (ganicke) [reply docs/behind-the-makecode-hardware.md]: Right, I should remember that. I think I will revert back this back. This is something we could fix in the render -- if imagUrl is defined don't show video frame.

File: docs/reference/input/acceleration.md (hunk 2 above)
Comment (ganicke) [reply docs/reference/input/acceleration.md]: Ok, it'll candy up the page more.

**Pull request merged**